fitbenchmarking/HTML_templates/table.css

```diff
@@ -0,0 +1,19 @@
+table {
+  border-collapse: collapse;
+}
+table, th, td {
+  border: 1px solid black;
+  height: min-content;
```

Comment:
I played around a bit more and found that setting the following fixes the cell height issue:
```
table, th, td {
  border: 1px solid black;
  width: auto;
  overflow: hidden;
}
th a, td a {
  display: block;
  text-decoration: none;
  color: rgb(30, 30, 30);
  margin-top: -10em;
  margin-bottom: -10em;
  padding-top: 10em;
  padding-bottom: 10em;
}
```